searx/engines/google_news.py

```diff
@@ -16,74 +45,155 @@
     "use_official_api": False,
     "require_api_key": False,
     "results": 'HTML',
+    "parse": ('url', 'title', 'img_src', 'content'),
```

Comment:
my take is `parse` is useless : 
* the code  between 187 and 192 is straightforward.
* from the user point of view, I don't know how it is relevant (if it is shown in the /preferences for example or the documentation).
* worst case scenario, this variable is not updated when the code is.
* writing this comment, I release that the same comment is valid for `results`.

See PR #2460

Reply:
Thanks for your remarks .. I just wanted to have a discussion, if it make sense to define a _interface_ .. but with your remarks in mind I also think _the code between 187 and 192 is straightforward._.

What I sometimes miss is a documentation of what parsed values are required by searx core (e.g. title) or supported by the templates ...  for me it is also not clear if it is OK to embed HTML in the _content_  or any other returned value:

https://github.com/searx/searx/blob/f310305c54c3cd1d9fc74f09453294edbd2b5486/searx/engines/youtube_api.py#L31-L33

Reply:
There is dedicated field for embedded media, see :
https://github.com/searx/searx/blob/f310305c54c3cd1d9fc74f09453294edbd2b5486/searx/engines/youtube_noapi.py#L80-L87

Reply:
> There is dedicated field for embedded media

Yes but only in the videos.html template ..  so I stay with:

> What I sometimes miss is a documentation of what parsed values are required by searx core (e.g. title) or supported by the templates.

Does it make sense to add an issue? Title:  _missing template documentation_

Or do you think it will be useless for some reason I don't see right now.